wetlab/utils/sample.py

```diff
@@ -255,7 +202,7 @@ def get_sample_in_project_obj_from_id(sample_in_project_id):
     return sample_in_project_obj
 
 
-def get_sample_in_project_obj_from_sample_name(sample_name_in_project):
+def get_sample_in_project_obj(sample_name_in_project):
```

Comment:
I understand that name is quite long, but renaming  the function like this it could create a little bite confusion because the existing "get_sample_in_project_obj_from_id". 
I think is better get the name as you type , but deleting function "get_sample_in_project_obj_from_id" and have only one which identify if input is name or primary key to return the object

Reply:
Ok, I'll try to merge both functions into one in the next PR